File: python/program/MobilBbm.py
from Mobil import Mobil

class MobilBbm(Mobil):
    def __init__(self, merk, nama, warna, tahun, transmisi, jumlahroda, mesin, ban, velg, kapasitasTangki, bahanBakar):
        super().__init__(merk, nama, warna, tahun, transmisi, jumlahroda, mesin, ban, velg)
        self.__kapasitasTangki = kapasitasTangki
        self.__bahanBakar = bahanBakar
    
    def set_kapasitasTangki(self, kapasitasTangki):
        self.__kapasitasTangki = kapasitasTangki
    
    def get_kapasitasTangki(self):
        return self.__kapasitasTangki
    
    def set_bahanBakar(self, bahanBakar):
        self.__bahanBakar = bahanBakar
    
    def get_bahanBakar(self):
        return self.__bahanBakar

File: Javaaa/program/MobilBbm.java
import java.util.List;

class MobilBbm extends Mobil {
    private int kapasitasTangki;
    private String bahanBakar;

    public MobilBbm() {
        super();
    }

    public MobilBbm(String merk, String nama, String warna, int tahun, String transmisi, int jumlahRoda, Mesin mesin, List<Ban> ban, List<Velg> velg, int kapasitasTangki, String bahanBakar) {
        super(merk, nama, warna, tahun, transmisi, jumlahRoda, mesin, ban, velg);
        this.kapasitasTangki = kapasitasTangki;
        this.bahanBakar = bahanBakar;
    }

    public void setKapasitasTangki(int kapasitasTangki) {
        this.kapasitasTangki = kapasitasTangki;
    }

    public int getKapasitasTangki() {
        return kapasitasTangki;
    }

    public void setBahanBakar(String bahanBakar) {
        this.bahanBakar = bahanBakar;
    }

    public String getBahanBakar() {
        return bahanBakar;
    }
}